projects/clearpool/index.js

```diff
@@ -1,53 +1,109 @@
 //  npm i -f
 //  node test.js projects/clearpool/index.js
-
+const axios = require("axios");
 const abi = require("./abi.json");
 
 const { stakings } = require("../helper/staking");
 const { getLogs } = require("../helper/cache/getLogs");
 
 const singleStakingContracts = ["0x629E39da1Db5654fe59cAE31d48CAEBB8dC2A9c6"];
 const CPOOL = "0x66761fa41377003622aee3c7675fc7b5c1c2fac5";
+const api = "https://squid.subsquid.io/cpool-squid/v/v1/graphql";
+
 
 module.exports = {
   methodology: "We count liquidity by USDC deposited on the pools contracts",
-}
+};
 
 const config = {
-  ethereum: { factory: '0xde204e5a060ba5d3b63c7a4099712959114c2d48', fromBlock: 14443222 },
-  polygon_zkevm: { factory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16', fromBlock: 302970 },
-  polygon: { factory: '0x215CCa938dF02c9814BE2D39A285B941FbdA79bA', fromBlock: 31128013 },
-  optimism: { factory: '0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80', fromBlock: 107128813 },
-}
+  ethereum: {
+    factory: "0xde204e5a060ba5d3b63c7a4099712959114c2d48",
+    fromBlock: 14443222,
+  },
+  polygon_zkevm: {
+    factory: "0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16",
+    fromBlock: 302970,
+  },
+  polygon: {
+    factory: "0x215CCa938dF02c9814BE2D39A285B941FbdA79bA",
+    fromBlock: 31128013,
+  },
+  optimism: {
+    factory: "0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80",
+    fromBlock: 107128813,
+  },
+};
 
-Object.keys(config).forEach(chain => {
-  const { fromBlock, factory } = config[chain]
+const getInfo = async () => {
```

Comment:
bit confused by this PR, I dont see that function used anywhere. other than that, there is only formatting changes?

Reply:
@g1nt0ki sry for confusion, we will mark the PR as ready once finished.